.github/workflows/test.yml

```diff
@@ -68,3 +68,26 @@ jobs:
         run: yarn workspace @thunderstore/nextjs build
       - name: Test @thunderstore/nextjs
         run: yarn workspace @thunderstore/nextjs jest
+
+  deploy:
+    name: Trigger deploy
```

Comment:
This could maybe mention that the deployment target is test environment? It might not be clear for reader even thought the file name points to that. 🤷🏻‍♂️